### PR TITLE
Add FWHM to aperture settings, and change name of settings object

### LINKED ISSUE #254: Add `fwhm` property to `ApertureSettings`?
I've been looking at how to condense the arguments to the photometry routines and was thinking about adding a FWHM property to `ApertureSettings`. I wanted to get some feedback before doing it though.

In favor of adding FWHM:

- We already calculate FWHM as part of calculating the aperture radius
- Aperture size is always related to FWHM

Against adding FWHM:

- I don't usually think of FWHM as an aperture setting. 

Thoughts, @JuanCab?

### PR DESCRIPTION
**Pull request by mwcraig**
This pull request ends up doing a few things, and may be easier to review commit-by-commit.

+ Adds a `fwhm` property to `ApertureSettings`, which fixes #254 
+ Refactor tests in `test_photometry` to use a single `ApertureSettings` object to make the transition to having the `fwhm` property easier
+ Configure `ApertureSettings` so that misspellings of properties will raise an error instead of silently using the default value.
+ Change `ApertureSettings` to `PhotometryApertures`
+ Fix an incorrect creation of `PhotometrySettings` in `aij_plots`.
+ Change photometry functions to use the `fwhm` property of `PhotometryApertures` instead of having a `fwhm` function argument.

### CODE REVIEW

File: stellarphot/gui_tools/seeing_profile_functions.py [hunk 1]
```diff
@@ -392,9 +392,9 @@ def show_event(
             else:
                 # User changed aperture
                 aperture_radius = aperture["radius"]
-                ap_settings = ApertureSettings(
+                ap_settings = PhotometryApertures(
                     **aperture
-                )  # Make an ApertureSettings object
+                )  # Make an aperture settings object, but don't update it's widget yet.
```
Comment (JuanCab) [inline stellarphot/gui_tools/seeing_profile_functions.py:397]: Maybe update this to be clear its an aperture photometry settings related thing.

File: stellarphot/plotting/aij_plots.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def seeing_plot(
     file_name : optional, string
         if entered, file will save as png with this name
 
-    aperture_settings : optional, `stellarphot.settings.ApertureSettings`
+    aperture_settings : optional, `stellarphot.settings.PhotometryApertures`
         The aperture settings used to create the plot. If not provided, the
         aperture radius will be set to 4 * HWHM, the inner annulus will be set
         to radius + 10, and the outer annulus will be set to radius + 25.
```
Comment (JuanCab) [inline stellarphot/plotting/aij_plots.py:49]: You probably want to update docs to indicate the default FWHM also, and rename the variable from `aperture_settings` (sorry!)

File: stellarphot/plotting/aij_plots.py [hunk 3]
```diff
@@ -59,8 +59,11 @@ def seeing_plot(
     """
     if aperture_settings is None:
         radius = 4 * HWHM
-        aperture_settings = ApertureSettings(
-            radius=radius, inner_annulus=radius + 10, outer_annulus=radius + 25
+        aperture_settings = PhotometryApertures(
```
Comment (JuanCab) [inline stellarphot/plotting/aij_plots.py:62]: Again, I suggest renaming the variable too!

File: stellarphot/settings/models.py [hunk 4]
```diff
@@ -234,21 +237,26 @@ class ApertureSettings(BaseModel):
...
     --------
 
-    To create an `ApertureSettings` object, you can pass in the radius, gap,
+    To create an `PhotometryApertures` object, you can pass in the radius, gap,
     and annulus_width as keyword arguments:
 
-    >>> aperture_settings = ApertureSettings(radius=4,
-    ...                                      gap=10,
-    ...                                      annulus_width=15)
+    >>> aperture_settings = PhotometryApertures(
```
Comment (JuanCab) [inline stellarphot/settings/models.py:243]: Update variable name `aperture_settings`?

**Review by JuanCab — COMMENTED** [review]
I suggested a few tweaks to the docs and a change of variable name.

**Comment by mwcraig** [comment]
I think I caught all the name changes...

File: stellarphot/photometry/photometry.py
Comment (JuanCab) [inline stellarphot/photometry/photometry.py:1]: Looks good with the name change.

**Review by JuanCab — APPROVED** [review]

**Pull request merged**